py/server/deephaven/table_listener.py

```diff
@@ -579,22 +575,20 @@ def stop(self) -> None:
             self.started = False
 
 
-def merged_listen(listener_recorders: Sequence[ListenerRecorder], listener: Union[Callable, MergedListener],
+def merged_listen(tables: Sequence[Table], listener: Union[Callable[[Dict[Table, TableUpdate]], None], MergedListener],
```

Comment:
We probably want to support extra deps for the merged listener itself.

Reply:
We do. The last parameter of the function.
``` python
def merged_listen(tables: Sequence[Table], listener: Union[Callable[[Dict[Table, TableUpdate]], None], MergedListener],
                  description: str = None, dependencies: Union[Table, Sequence[Table]] = None) -> MergedListenerHandle:

```
Is that what you are looking for?